newrelic-agent/src/main/java/com/newrelic/agent/Transaction.java

```diff
@@ -2562,4 +2562,8 @@ int getCountOfRunningAndFinishedTransactionActivities() {
             return runningChildren.size() + finishedChildren.size();
         }
     }
+
+    public Object getSecurityMetaData() {
+        return null;
```

Comment:
This differs a little from the implementation on the k2-integration branch.
In there this method returns a SecurityMetaData. But it would require that we have a dependency on the csec api.
So for now, this implementation only returns null. Since nothing uses this method yet, this should be fine.